e2e/updatecli.d/success.d/toolversions.yaml

```diff
@@ -0,0 +1,36 @@
+name: Basic .tool-versions Example
```

Comment:
I'm trying to test this out now:

```bash
dist/updatecli_darwin_arm64/updatecli --debug diff --config e2e/updatecli.d/success.d/toolversions.yaml
```

but it failed with:

```

################################
# BASIC .TOOL-VERSIONS EXAMPLE #
################################


SOURCES
=======

local
-----
ERROR: ✗ ⚠ Don't support resource kind: toolversions
Pipeline "Basic .tool-versions Example" failed
Skipping due to:
	sources stage:	"⚠ Don't support resource kind: toolversions"

```

Reply:
You also need to add toolversions to [GetResourceMapping](https://github.com/updatecli/updatecli/blob/f156b7b836b37489bb32b8eccbe1f8ae1c72e4e3/pkg/core/pipeline/resource/main.go#L239)

Reply:
Thanks! It works like a charm now:

```bash
 dist/updatecli_darwin_arm64/updatecli --debug diff --config e2e/updatecli.d/success.d/toolversions.yaml


+++++++++++
+ PREPARE +
+++++++++++

Loading Pipeline "e2e/updatecli.d/success.d/toolversions.yaml"
DEBUG: pipelineid undefined, we'll try to generate one
DEBUG: using pipeline name to generate the pipelineid

SCM repository retrieved: 0


++++++++++++++++++
+ AUTO DISCOVERY +
++++++++++++++++++



++++++++++++
+ PIPELINE +
++++++++++++



################################
# BASIC .TOOL-VERSIONS EXAMPLE #
################################


SOURCES
=======

local
-----
✔ value "1.0.0", found in file "/Users/vmartinez/workspaces/elastic/updatecli/pkg/plugins/resources/toolversions/testdata/.tool-versions", for key "bats"'
DEBUG: empty changelog found for the source


CONDITIONS:
===========

local
-----
✔ key "bats", from file "pkg/plugins/resources/toolversions/testdata/.tool-versions", is correctly set to "1.0.0"

golang
------
✔ key "golang", from file "pkg/plugins/resources/toolversions/testdata/.tool-versions", is correctly set to "1.8.2"


TARGETS
========

local-source
------------

**Dry Run enabled**

⚠ - key "tool", from file "pkg/plugins/resources/toolversions/testdata/.tool-versions", is incorrectly set to "" and should be "1.0.0"

local-value
-----------

**Dry Run enabled**

⚠ - key "new-tool", from file "pkg/plugins/resources/toolversions/testdata/.tool-versions", is incorrectly set to "" and should be "2.0.0"


ACTIONS
========


=============================

REPORTS:



⚠ Basic .tool-versions Example:
	Source:
		✔ [local] Get value from .tool-versions
	Condition:
		✔ [golang] Test value from .tool-versions
		✔ [local] Test value from .tool-versions with sourceid
	Target:
		⚠ [local-source] Test sourceid new key from .tool-versions
		⚠ [local-value] Test new tool with value from .tool-versions


Run Summary
===========
Pipeline(s) run:
  * Changed:	1
  * Failed:	0
  * Skipped:	0
  * Succeeded:	0
  * Total:	1
```